jina/orchestrate/deployments/config/k8slib/kubernetes_deployment.py

```diff
@@ -54,6 +56,7 @@ def get_deployment_yamls(
     :param monitoring: enable monitoring on the deployment
     :param port_monitoring: port which will be exposed, for the prometheus server, by the deployed containers
     :param protocol: In case of being a Gateway, the protocol used to expose its server
+    :param volumes: If volumes are passed to Executors, we will try to dump to StatefulSet
```

Comment:
```suggestion
    :param volumes: If volumes are passed to Executors, Jina will create a StatefulSet instead of Deployment and include the first volume in the volume mounts
```